src/containers/AccountConnection.jsx

```diff
@@ -125,7 +124,7 @@ class AccountConnection extends Component {
             const currentIdx = accounts.findIndex(a => a._id === accountID)
             const account = accounts[currentIdx]
             this.setState({account: account})
-            account.folderPath = account.folderPath || t('account.config.default_folder', konnector)
+            if (folderPath) account.folderPath = folderPath
```

Comment:
It's to get default folderPath here if it's doesn't exist. But the check is handled by `getFolderPathIfNecessary` called to get folderPath here.